src/gui/render.cpp

```diff
@@ -623,26 +623,34 @@ static void ChangeScaler(bool pressed) {
 	RENDER_CallBack( GFX_CallBackReset );
 } */
 
+const std::unordered_map<std::string, const char *> builtin_shaders = {
```

Comment:
Would it also benefit plain linear output by adding `none` to the list?

Reply:
Yes, probably, but it should rather be done where the later checks are as `none` is not a shader, right?

Reply:
Done.